client/gutenberg/extensions/publicize/store/controls.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Trigger an API Fetch request.
+ *
+ * @param {Object} action Action Object.
+ *
+ * @return {Promise} Fetch request promise.
+ */
+const fetchFromApi = action => {
+	return apiFetch( { path: action.path } );
```

Comment:
Could destructure the `action` on L13 for moar noice

Reply:
Good call! Done in b584d8e49f4ba2637811b40e27225856b9cc78de.